addons/web/static/tests/search/search_bar_tests.js

```diff
@@ -898,4 +902,50 @@ QUnit.module("Search", (hooks) => {
         await triggerEvent(searchInput, null, "keydown", { key: "ArrowUp" });
         assert.containsOnce(target, ".focus");
     });
+
+    QUnit.test("check kwargs of a rpc call with a domain", async function (assert) {
+        assert.expect(3);
+
+        const mockRPC = async (route, args) => {
+            if (route.includes("/partner/name_search")) {
+                assert.deepEqual(args, {
+                    "model": "partner",
```

Comment:
" are not necessary for object keys
--> maybe you should use the tools in addons/web/tooling when you work in web?

Reply:
Thank you for the fix

Reply:
I was not aware of addons/web/tooling so maybe thle is not too

the documentation seems to be in https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines and after installing it (by running enable.sh), the pre-commit format the change and remove " around keys.